rfcs/text/0000-host-and-hostname-fields.md

```diff
@@ -0,0 +1,112 @@
+# 0000: host and hostname fields
+<!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
+
+- Stage: **0 (strawperson)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Date: **TBD** <!-- The ECS team sets this date at merge time. This is the date of the latest stage advancement. -->
+
+<!--
+As you work on your RFC, use the "Stage N" comments to guide you in what you should focus on, for the stage you're targeting.
+Feel free to remove these comments as you go along.
+-->
+
+<!--
+Stage 0: Provide a high level summary of the premise of these changes. Briefly describe the nature, purpose, and impact of the changes. ~2-5 sentences.
+-->
+
+Include host information (serial number, manufacturer, and model), bring hostname across other applicable fieldsets.
```

Comment:
This summary is sparse.

Can you describe the motivation, purpose, and/or impact of these proposed changes a bit more?

Reply:
These fields represent fields not currently represented in the ECS standard that we see in our data sources Tenable SC, Tanium, etc.) that we believe are important enough to create an ECS field to capture for our ELK Siem users.

Reply:
These fields represent fields not currently represented in the ECS standard that we see in our data sources Tenable SC, Tanium, etc.) that we believe are important enough to create an ECS field to capture for our ELK Siem users.

Reply:
Thanks for the context. Can you capture that in the proposal doc?

Reply:
Done